### PR TITLE
Make the Linux Server guide less SSH-centric

### PR DESCRIPTION
**Pull request by ptgott**
The current Linux Server guide includes instructions for accessing the server and viewing a session recording. For users who are not interested in server access, this can be irrelevant or even a turnoff. Instead of showing SSH-specific features, make the final step about registering resources using the Web UI.

### CODE REVIEW

**Comment by ptgott** [comment]
@rodgtr1 I incorporated some of your feedback in 6dd94ef7a4c834d07301f7523656d97ebabd10b0 .

The feedback I haven't incorporated:
- I wasn't sure how to use the term "proxy" more in the guide (other than "Proxy Service", of course)
- We'll still need to find a good diagram to use. [This PR](https://github.com/gravitational/teleport/pull/25730/files) that I opened before last Friday's meeting adds the following Mermaid diagram, though it will probably need tweaking/styling if we're giving it pride of place. (We can also have someone with graphic design talent create something):

```mermaid
flowchart LR
subgraph private[Private network]
    direction LR
    serv1[Teleport service]
    serv2[Teleport service]
    serv3[Teleport service]
end
subgraph public[Public network]
    Developers
end
subgraph server[Linux server]
    direction RL
    auth["Auth Service"]
    proxy[Proxy Service]
    ssh[SSH Service]
    auth-- "Short-lived credentials" -->proxy
    ssh-- Reverse tunnel-->proxy
end
Developers-- tsh sessions --->server
serv1 & serv2 & serv3-- Reverse tunnel--->server
```

- It leaves the Linux Server guide in `/docs/get-started`. I'll open a separate draft PR with the beginnings of a change to make the Linux Server guide the landing page.

File: docs/pages/get-started.mdx [hunk 1]
```diff
@@ -212,81 +209,45 @@ $ tsh login --proxy=<Var name="teleport.example.com" /> --user=teleport-admin
...
+behind a single identity-aware access proxy, including servers, databases,
+applications, Kubernetes clusters, Windows desktops, and cloud provider APIs.
 
-In the Teleport Web UI, click the dropdown menu on the upper left of the screen,
-then choose **Management**. click the **Activity** tab in the sidebar on the
-left of the screen, then click **Session Recordings**. You will see your session
-from the previous step:
+To enroll a resource with Teleport, visit the Web UI and click the name of a
+resource on the sidebar. The Web UI will show you the steps you can take to
+enroll that resource.
```
Comment (alexfornuto) [inline docs/pages/get-started.mdx:223]: Can we make this more explicit by referencing the **Add `$RESOURCE`** button(s)?

**Review by alexfornuto — APPROVED** [review]

**Comment by rodgtr1** [comment]
@ptgott  I think this PR is perfect, with the addition of Alex's suggestions. As for diagrams, what about something like this (would get Peter to make it look better). Or is this too basic, i.e. some more details added? Trying to keep it on par with the guide. 

image

**Comment by rodgtr1** [comment]
> @rodgtr1 What would you think of us working on the diagram in a subsequent PR? That way we can merge these easy-to-make changes and see what needs to be done in terms of getting a nice diagram together?

Yes, lets do that in another PR

**Comment by ptgott** [comment]
@rodgtr1 Sounds good! Is there anything else we should do in this PR before merging?

**Comment by rodgtr1** [comment]
> @rodgtr1 Sounds good! Is there anything else we should do in this PR before merging?

@ptgott No I think this is sufficient for now. We can mull over it for a bit and then create a new PR for any further tweaks.

**Comment by ptgott** [comment]
@rodgtr1 👍 Do you have permissions to approve?

**Review by rodgtr1 — APPROVED** [review]

**Comment by rodgtr1** [comment]
> @rodgtr1 👍 Do you have permissions to approve?

Ahh, though I did. Done.

**Pull request merged**